action.yml

```diff
@@ -60,25 +64,32 @@ runs:
         echo "::set-output name=new_tag::$NEW_TAG"
       shell: bash
 
+    - name: Use overridden tag if supplied
+      id: tag_resolution
+      run: |
+        TAG=${{ steps.tag_generation.outputs.new_tag }}${{ inputs.version-override }}
```

Comment:
Do we just append an override to the end of the generated tag here?

Reply:
If override is not empty then no tag will be generated and the previous step will be skipped. Therefore we're appending override to an empty string.
Otherwise, if no override is provided, tag will be generated and prepended to an empty string of an override.